app/models/ability.rb

```diff
@@ -87,9 +88,8 @@ def signed_in(user)
     end
 
     # cannot create an event if program does not have open cfp
-    cannot [:new, :create], Event do |event|
-      user_inclusion = event.event_users.map { |event_user| event_user.user.id }.compact.include? user.id
-      !event.program.cfp_open? || !event.new_record? || !user_inclusion
+    can [:new, :create], Event do |event|
```

Comment:
This is included in https://github.com/ChrisBr/osem/blob/fix_ability/app/models/ability.rb#L86
```
can :manage, Event do |event|
  event.users.include?(user)
end
```

The extra thing we want is to supersede that `can :manage` and disallow access (cannot) if the cfp is closed.